spec/presenters/sufia/collection_presenter_spec.rb

```diff
@@ -58,4 +58,9 @@
     subject { presenter.total_items }
     it { is_expected.to eq 0 }
   end
+
+  describe "#identifier" do
+    subject { presenter.identifier }
```

Comment:
Will this always be nil? Or is it a function of the delegation?

https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/independent/delegate_method_matcher.rb

` it { is_expected.to delegate_method(:identifier).to(:solr_document) }`